platalea/basic.py

```diff
@@ -85,7 +90,7 @@ def val_loss():
         for epoch in range(1, config['epochs']+1):
             cost = Counter()
             for j, item in enumerate(data['train'], start=1): # check reshuffling
-                item = {key: value.cuda() for key, value in item.items()}
+                item = {key: value.to(_device) for key, value in item.items()}
                 loss = net.cost(item)
                 optimizer.zero_grad()
                 loss.backward()
```

Comment:
Using device should be done everywhere `cuda()` is used to be consistent.

Reply:
Ah sure, weird that i didn't catch that one before.